doc/source/howtos/proxysql.rst

```diff
@@ -500,28 +491,26 @@ which is monitored by ProxySQL and can be set to one of the following values:
 * ``DISABLED``: This is the default state
   that tells ProxySQL to route traffic to the node as usual.
 
-* ``SHUTDOWN``: This state is set automatically
-  when you initiate node shutdown.
+* ``SHUTDOWN``: This state is set 
+  when you initiate node shutdown automatically.
```

Comment:
Hmm. I think this modification would change the semantics of this sentence. It's the *state* that is set automatically on shutdown. Now it reads as if the node shutdown is initiated automatically.

Reply:
Reverted